src/Files.App/Views/LayoutModes/ColumnViewBrowser.xaml

```diff
@@ -8,7 +8,6 @@
 	xmlns:icore="using:Microsoft.Xaml.Interactions.Core"
 	xmlns:local="using:Files.App"
 	xmlns:mc="http://schemas.openxmlformats.org/markup-compatibility/2006"
-	x:Name="PageRoot"
```

Comment:
Are you sure this isn't being used in an item template?

Reply:
I'm sorry. Fixed.